bindings/rust/wasmedge-sys/src/config.rs

```diff
@@ -0,0 +1,152 @@
+use super::wasmedge;
+pub struct Config {
+    pub(crate) ctx: *mut wasmedge::WasmEdge_ConfigureContext,
+    _private: (),
```

Comment:
Since `wasmedge-sys` is also a standalone crate, if someone else wants to rely on it to implement a third-party sdk, then we expect that the third-party sdk will only use the interface we want it to use.